opus/import/obs_instrument_ebrocc.py

```diff
@@ -111,38 +120,51 @@ def field_obs_wavelength_wavelength2(self):
     ################################
 
     def field_obs_profile_occ_type(self):
-        return 'STE'
+        return self._create_mult('STE')
 
     def field_obs_profile_occ_dir(self):
         occ_dir = self._index_col('OCCULTATION_DIRECTION')
         if occ_dir in ('INGRESS', 'EGRESS', 'BOTH'):
-            return occ_dir[0]
+            return self._create_mult(occ_dir[0])
         self._log_nonrepeating_error(f'Unknown OCCULTATION_DIRECTION "{occ_dir}"')
-        return None
+        return self._create_mult(None)
 
     def field_obs_profile_body_occ_flag(self):
-        return self._supp_index_col('PLANETARY_OCCULTATION_FLAG')
+        body_occ_flag = self._supp_index_col('PLANETARY_OCCULTATION_FLAG')
+        return self._create_mult(body_occ_flag)
 
     def field_obs_profile_quality_score(self):
-        return ("UNASSIGNED", "Unassigned")
+        data_dict = self._create_mult(
+            col_val='UNASSIGNED',
+            disp_name='Unassigned',
+        )
+        return data_dict
 
     def field_obs_profile_wl_band(self):
         wl = self._supp_index_col('WAVELENGTH') # microns
         if wl > 0.7:
-            return 'IR'
+            return self._create_mult('IR')
         if wl > 0.4:
-            return 'VIS'
-        return 'UV'
+            return self._create_mult('VIS')
+        return self._create_mult('UV')
 
     def field_obs_profile_source(self):
         target_name, target_info = self._star_name_helper('index_label', 'STAR_NAME')
         if target_info is None:
-            return None
-        return target_name, target_info[2]
+            return self._create_mult(None)
+        data_dict = self._create_mult(
+            col_val=target_name,
+            disp_name=target_info[2]
+        )
+        return data_dict
 
     def field_obs_profile_host(self):
         insthost = self._supp_index_col('INSTRUMENT_HOST_NAME')
-        return (insthost, insthost)
+        data_dict = self._create_mult(
+            col_val=insthost,
```

Comment:
This would be a good place to have something like self._create_mult_keep_case(insthost)